app/controllers/users_controller.rb

```diff
@@ -1,6 +1,7 @@
 class UsersController < ApplicationController
   def index
     @users = User.all
+    # render json: @users
   end
 
   def show
```

Comment:
- Kindly make sure that all API endpoints receive JSON and respond in a JSON format . as I see here your comment on the code that must return the JSON format please fix that :+1:  and make sure that you create the endpoint to return the list of users and is working as expected . to do that I can suggest you create a separate API file in your control for all and point :+1: